packages/adapter-cloudflare/index.js

```diff
@@ -60,26 +60,35 @@ export default function () {
 /**
  * @param {import('@sveltejs/kit').Builder} builder
  * @param {string[]} assets
+ * @param {import('./index').AdapterOptions} options
  * @returns {import('.').RoutesJSONSpec}
  */
-function get_routes_json(builder, assets) {
+function get_routes_json(builder, assets, options) {
+	let {
+		autoGenerate = true,
+		include = [],
+		exclude = []
+	} = options?.routes ?? /** @type {import('./index').AdapterOptions} */ ({ routes: {} }).routes;
```

Comment:
```suggestion
/** @type {import('./index').AdapterOptions['routes']} */
let {
	autoGenerate = true,
	include = [],
	exclude = []
} = options?.routes ?? {};
```

Reply:
Did you remove indentation? Or is that GitHub showing it incorrectly?

Reply:
fixed I think